web/packages/teleport/src/TopBar/TopBar.tsx

```diff
@@ -335,22 +321,32 @@ const MainNavItem = ({
   isSelected: boolean;
   to: string;
   name: string;
-  Icon: (props: { color: string }) => JSX.Element;
+  Icon: (props: { color: string; size: number }) => JSX.Element;
 }) => {
+  const { currentWidth } = useLayout();
+  const theme: Theme = useTheme();
+  const mediumAndUp = currentWidth >= theme.breakpoints.medium;
   return (
-    <NavigationButton selected={isSelected} to={to} title={name}>
-      <Icon color={isSelected ? 'text.main' : 'text.muted'} />
+    <NavigationButton
+      selected={isSelected}
+      to={to}
+      title={!mediumAndUp ? name : ''}
+    >
+      <Icon
+        color={isSelected ? 'text.main' : 'text.muted'}
+        size={mediumAndUp ? 24 : 20}
+      />
       <Text
         ml={3}
         fontSize={18}
```

Comment:
Didn't we want to decrease the font size?
```suggestion
        fontSize={3}
```

Reply:
You are right, but only below 1280px. I've updated